packages/good-design/src/utils/string.ts

```diff
@@ -0,0 +1,6 @@
+export const capitalizeFirstLetter = (str: string) => {
```

Comment:
why not use styles for that?

Reply:
I am not sure I understand. what do you mean 'use styles' ?

Reply:
good question. no-idea why I did not think of it. changed now